static/.htaccess

```diff
@@ -149,6 +149,19 @@ RewriteRule ^optimize/3.11.0/self-managed/(.*)$ /docs/8.3/self-managed/$1 [R=301
 #   8.8: content moves introduced prior to the release of version 8.8.
 #---------------------------------------------------------------------------------
 
+# Rename C8 API
+RewriteRule ^docs/next/apis-tools/camunda-api-rest/(.*)$ /docs/next/apis-tools/orchestration-cluster-api-rest/$1 [R=301,L]
+RewriteRule ^docs/next/apis-tools/camunda-api-rest/camunda-api-rest-authentication/?$ /docs/next/
+apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-authentication/ [R=301,L]
+RewriteRule ^docs/next/apis-tools/camunda-api-rest/camunda-api-rest-data-fetching/?$ /docs/next/
+apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-data-fetching/ [R=301,L]
+RewriteRule ^docs/next/apis-tools/camunda-api-rest/camunda-api-rest-overview/?$ /docs/next/
+apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview/ [R=301,L]
+RewriteRule ^docs/next/apis-tools/camunda-api-rest/camunda-8-api-intermediate-tutorial/?$ /docs/next/
+apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-intermediate-tutorial/ [R=301,L]
+RewriteRule ^docs/next/apis-tools/camunda-api-rest/camunda-8-api-tutorial/?$ /docs/next/
+apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-tutorial/ [R=301,L]
```

Comment:
Do we have unintentional line breaks in here? Or is this just a weird view? I believe there are line breaks here we don't want 😅

Reply:
You are correct 😄 Adjusted.